js/poloniexfutures.js

```diff
@@ -0,0 +1,1636 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const { BadRequest, ArgumentsRequired, InvalidOrder, AuthenticationError, NotSupported, RateLimitExceeded, ExchangeNotAvailable, InvalidNonce, AccountSuspended, OrderNotFound } = require ('./base/errors');
+const Precise = require ('./base/Precise');
+const Exchange = require ('./base/Exchange');
+const { TICK_SIZE } = require ('./base/functions/number');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class poloniexfutures extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'poloniexfutures',
+            'name': 'Poloniex Futures',
+            'countries': [ 'US' ],
+            // 3 requests per second for some unauthenticated market endpoints => 1000ms / 3 = 333ms between requests
+            'rateLimit': 333,
```

Comment:
@AlgoQ the RL value and weights are off, I think

The "lightest" endpoint allows 300 req / 10 sec --> 30 req per second, so RL would be 1000/30 = 33.333 (one req every 33ms = 1 cost)? 

So for order handling, we can fire 200 req/ 10 sec --> 20 req per second, 1 req every 50ms ---> 50/33.333 = 1.5 cost ?  

![image](https://user-images.githubusercontent.com/43336371/210339488-c73c9dd9-b851-4a10-9410-e70d40dbf0f1.png)